packages/headless/src/features/triggers/triggers-state.ts

```diff
@@ -3,8 +3,14 @@ export interface TriggerState {
    * The URL to redirect the user to after receiving a redirection trigger.
    */
   redirectTo: string | null;
```

Comment:
https://github.com/coveo/ui-kit/pull/875/files#r654577518

Not really sure I get this, if we can have the query as an empty string initially, same for the redirectTo?

Reply:
Yes, I will open a separate PR to do a couple of fixes to the Redirect Triggers.